pkg/tracingpolicy/tracingpolicy.go

```diff
@@ -18,6 +18,10 @@ type TracingPolicy interface {
 	TpSpec() *v1alpha1.TracingPolicySpec
 	// TpInfo returns a description of the policy
 	TpInfo() string
+
+	TpHasAnnotation(string) bool
+
+	TpAnnotations() map[string]string
```

Comment:
As I said in the other comment, I don't see a use-case for this. Whatever we want we can add into `TpInfo()`. I'd like to keep the `TracingPolicy` interface as simple as possible.

Reply:
I have made the change to not display the author for now, so it is added as fields for now to allow loading only.